modules/virt-disable-auto-updates-single-boot-source.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assembly:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-automatic-bootsource-updates.adoc
+//
+
+:_content-type: PROCEDURE
+[id="virt-disable-auto-updates-single-boot-source_{context}"]
+= Disabling automatic updates for a single boot source
+
+You can disable automatic updates for an individual boot source, whether it is custom or system-defined, by editing the `HyperConverged` custom resource (CR).
+
+.Procedure
+
+. Open the `HyperConverged` CR in your default editor by running the following command:
++
+[source,terminal]
+----
+$ oc edit hyperconverged kubevirt-hyperconverged -n openshift-cnv
+----
+
+. Disable automatic updates for an individual boot source by editing the `spec.dataImportCronTemplates` field.
+
+* To disable automatic updates for a custom boot source, remove the boot source from the `spec.dataImportCronTemplates` field. Automatic updates are disabled for custom boot sources by default.
+
+* To disable automatic updates for a system-defined boot source:
+.. Add the boot source to `spec.dataImportCronTemplates`. 
++
+[NOTE]
+====
+Automatic updates are enabled by default for system-defined boot sources, but these boot sources are not listed in the CR unless you add them.
+====
+.. Set the value of the `dataimportcrontemplate.kubevirt.io/enable` annotation to `false`.
+
++
```

Comment:
This block is not attached to the previous step. I _think_ you can fix it by removing this line break:
```suggestion
.. Set the value of the `dataimportcrontemplate.kubevirt.io/enable` annotation to `false`.
+
```
I know these are a bit weird though - if it's misbehaving check this bit of [adoc reference](https://docs.asciidoctor.org/asciidoc/latest/lists/continuation/#enclose-in-open-block).

Reply:
No idea how that line break got in there! Great catch.